test_autogalaxy/profiles/light/shapelets/test_polar.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import division, print_function
+import numpy as np
+import pytest
+
+import autogalaxy as ag
+
+grid = np.array([[1.0, 1.0], [2.0, 2.0], [3.0, 3.0], [2.0, 4.0]])

```

Comment:
Slightly safer to define this as a fixture in case the array is mutated in a test